test/acceptance/keywords/cookies.robot

```diff
@@ -48,9 +49,16 @@ Get Cookies When There Are None
     ${cookies}=    Get Cookies
     Should Be Equal    ${cookies}    ${EMPTY}
 
+Get Cookie Expiry Set By Selenium
+    [Documentation]    Get Cookie Value Set By Selenium
+    [Tags]  Known Issue Firefox
+    ${cookie_dict}=    Get Cookie    another
+    ${expiry} =  Convert To Integer  1822148495
```

Comment:
One can not trust that expiry value is constant. Please set the cookie expiry first and then read it.

Reply:
I am setting the cookie expiry in the "add cookie" keyword in this file.  It is returning the same value consistently, so it seems like it is working in a similar manner to the existing tests, but I may not understand what you are asking for in this comment.